### PR TITLE
feat(dev): add Ansible playbook for ComfyStream setup

### PR DESCRIPTION
**Pull request by rickstaa**
This pull request adds an Ansible playbook in the `scripts` folder to automate ComfyStream setup on any cloud VM. It also changes the Caddy release from `test` to `stable` and prevents cloud-init from adding duplicate import lines.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -1,48 +1,102 @@
...
+
+5. **Run the Playbook**:  
+   Execute:
 
     ```bash
-    conda create -n comfystream python=3.8
-    conda activate comfystream
-    pip install -r requirements.txt
+    ansible-playbook -i ansible/inventory.yaml ansible/plays/setup_comfystream.yaml
     ```
```
Comment (eliteprox) [inline scripts/README.md:40]: If you're deploying with a non-sudo user account, it is necesary to use `--ask-become-pass` or configure a vault

**Review by eliteprox — COMMENTED** [review]
Tested ansible script and went pretty smooth. 


There were two errors that occured, everything still worked since I have these dependencies already:
```
TASK [Check if NVIDIA Container Toolkit is installed] ***************************************************************************************************************************************************************************************************************************
fatal: [comfystream_vm]: FAILED! => {"changed": false, "cmd": ["dpkg", "-l", "|", "grep", "nvidia-container-toolkit"], "delta": "0:00:00.011462", "end": "2025-03-04 15:28:15.072369", "msg": "non-zero return code", "rc": 1, "start": "2025-03-04 15:28:15.060907", "stderr": "dpkg-query: no packages found matching |", "stderr_lines": ["dpkg-query: no packages found matching |"], "stdout": "Desired=Unknown/Install/Remove/Purge/Hold\n| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend\n|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)\n||/ Name                     Version      Architecture Description\n+++-========================-============-============-=================================\nii  grep                     3.11-4build1 amd64        GNU grep, egrep and fgrep\nii  nvidia-container-toolkit 1.17.4-1     amd64        NVIDIA Container toolkit", "stdout_lines": ["Desired=Unknown/Install/Remove/Purge/Hold", "| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend", "|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)", "||/ Name                     Version      Architecture Description", "+++-========================-============-============-=================================", "ii  grep                     3.11-4build1 amd64        GNU grep, egrep and fgrep", "ii  nvidia-container-toolkit 1.17.4-1     amd64        NVIDIA Container toolkit"]}
...ignoring

TASK [Check if NVIDIA runtime is configured in Docker] **************************************************************************************************************************************************************************************************************************
fatal: [comfystream_vm]: FAILED! => {"changed": false, "cmd": ["docker", "info", "|", "grep", "-i", "nvidia"], "delta": "0:00:00.151926", "end": "2025-03-04 15:28:16.165888", "msg": "non-zero return code", "rc": 125, "start": "2025-03-04 15:28:16.013962", "stderr": "unknown shorthand flag: 'i' in -i\n\nUsage:  docker info [OPTIONS]\n\nRun 'docker info --help' for more information", "stderr_lines": ["unknown shorthand flag: 'i' in -i", "", "Usage:  docker info [OPTIONS]", "", "Run 'docker info --help' for more information"], "stdout": "", "stdout_lines": []}
...ignoring
```

**Comment by rickstaa** [comment]
> Tested ansible script and went pretty smooth.
> 
> There were two errors that occured, everything still worked since I have these dependencies already:
> 
> ```
> TASK [Check if NVIDIA Container Toolkit is installed] ***************************************************************************************************************************************************************************************************************************
> fatal: [comfystream_vm]: FAILED! => {"changed": false, "cmd": ["dpkg", "-l", "|", "grep", "nvidia-container-toolkit"], "delta": "0:00:00.011462", "end": "2025-03-04 15:28:15.072369", "msg": "non-zero return code", "rc": 1, "start": "2025-03-04 15:28:15.060907", "stderr": "dpkg-query: no packages found matching |", "stderr_lines": ["dpkg-query: no packages found matching |"], "stdout": "Desired=Unknown/Install/Remove/Purge/Hold\n| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend\n|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)\n||/ Name                     Version      Architecture Description\n+++-========================-============-============-=================================\nii  grep                     3.11-4build1 amd64        GNU grep, egrep and fgrep\nii  nvidia-container-toolkit 1.17.4-1     amd64        NVIDIA Container toolkit", "stdout_lines": ["Desired=Unknown/Install/Remove/Purge/Hold", "| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend", "|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)", "||/ Name                     Version      Architecture Description", "+++-========================-============-============-=================================", "ii  grep                     3.11-4build1 amd64        GNU grep, egrep and fgrep", "ii  nvidia-container-toolkit 1.17.4-1     amd64        NVIDIA Container toolkit"]}
> ...ignoring
> 
> TASK [Check if NVIDIA runtime is configured in Docker] **************************************************************************************************************************************************************************************************************************
> fatal: [comfystream_vm]: FAILED! => {"changed": false, "cmd": ["docker", "info", "|", "grep", "-i", "nvidia"], "delta": "0:00:00.151926", "end": "2025-03-04 15:28:16.165888", "msg": "non-zero return code", "rc": 125, "start": "2025-03-04 15:28:16.013962", "stderr": "unknown shorthand flag: 'i' in -i\n\nUsage:  docker info [OPTIONS]\n\nRun 'docker info --help' for more information", "stderr_lines": ["unknown shorthand flag: 'i' in -i", "", "Usage:  docker info [OPTIONS]", "", "Run 'docker info --help' for more information"], "stdout": "", "stdout_lines": []}
> ...ignoring
> ```

STrange maybe something broke in my last commit, where I made this check optional. I will try it tomorrow with new mcahine. It doesn't give a problem with excisting machines.

File: scripts/README.md (hunk 1 above)
Comment (rickstaa) [reply scripts/README.md:40]: Good point will update the readme.

**Review by eliteprox — APPROVED** [review]
LGTM!
